src/components/navigation-bar/navigation-bar.config.js

```diff
@@ -0,0 +1 @@
+export default ["top", "bottom"];
```

Comment:
It isn't clear what this is just looking at this file

You should `export const STICKY_POSITIONS = ["top", "bottom"];` and just disable the eslint rule that will show

Reply:
thanks, I've removed this file